src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx

```diff
@@ -471,6 +471,43 @@ Not every field listed in this glossary is required for every condition type. Th
     * Plugin conditions
   </Collapser>
 
+  <Collapser
+    id="signal_aggregation_delay"
+    title="signal[aggregation_window]"
+  >
+    The amount of time before aggregated data is evaluated. Default is **2 minutes**.
+
+    Used with Event flow and Cadence aggregation methods.
```

Comment:
think this should be sentence case, e.g., 'Used with event flow and cadence aggregation methods.' Or if there's a need to really emphasize them as types, maybe